Aquality.Selenium/src/Aquality.Selenium/Browsers/BrowserFactory.cs

```diff
@@ -1,14 +1,24 @@
-﻿namespace Aquality.Selenium.Browsers
+﻿using Aquality.Selenium.Core.Localization;
+
+namespace Aquality.Selenium.Browsers
 {
     /// <summary>
     /// Abstract representation of <see cref="IBrowserFactory"/>.
     /// </summary>
     public abstract class BrowserFactory : IBrowserFactory
     {
-        protected BrowserFactory()
+        protected BrowserFactory(ILocalizedLogger localizedLogger)
```

Comment:
Why only localizedlogger? Both factories have the same parameters.
new RemoteBrowserFactory(Get<IActionRetrier>(), Get<IBrowserProfile>(), Get<ITimeoutConfiguration>(), LocalizedLogger);
new LocalBrowserFactory(Get<IActionRetrier>(), Get<IBrowserProfile>(), Get<ITimeoutConfiguration>(), LocalizedLogger);

Reply:
Because only this parameter is used in this class.
Do I  really need to define parameters which are not used? There could be a different factory outside of this library...